refinedstorage2-platform-fabric/src/main/java/com/refinedmods/refinedstorage2/platform/fabric/integration/recipemod/rei/FilterGuiFocusStackProvider.java

```diff
@@ -0,0 +1,40 @@
+package com.refinedmods.refinedstorage2.platform.fabric.integration.recipemod.rei;
+
+import com.refinedmods.refinedstorage2.platform.api.integration.recipemod.IngredientConverter;
+import com.refinedmods.refinedstorage2.platform.api.resource.filter.FilteredResource;
+import com.refinedmods.refinedstorage2.platform.common.containermenu.AbstractResourceFilterContainerMenu;
+import com.refinedmods.refinedstorage2.platform.common.containermenu.slot.ResourceFilterSlot;
+import com.refinedmods.refinedstorage2.platform.common.screen.AbstractBaseScreen;
+
+import dev.architectury.event.CompoundEventResult;
+import me.shedaniel.math.Point;
+import me.shedaniel.rei.api.client.registry.screen.FocusedStackProvider;
+import me.shedaniel.rei.api.common.entry.EntryStack;
+import net.minecraft.client.gui.screens.Screen;
+
+public class FilterGuiFocusStackProvider implements FocusedStackProvider {
```

Comment:
Why suddenly introduce the term "gui" here? 
```suggestion
public class FilteredResourceFocusedStackProvider implements FocusedStackProvider {
```